services/app-web/src/components/Modal/Modal.tsx

```diff
@@ -66,7 +66,7 @@ export const Modal = ({
                         onClick={onSubmit}
                         {...submitButtonProps}
                     >
-                        Submit
+                        {onSubmitText || 'Submit'}
```

Comment:
Can you add a test to `Modal.test.tsx` for this new prop that was added? That way we have test coverage for this new logic